src/components/TextInput.tsx

```diff
@@ -0,0 +1,197 @@
+import React, {useEffect, useState} from "react";
+import {
+    Keyboard,
+    KeyboardTypeOptions,
+    StyleProp,
+    StyleSheet,
+    Text,
+    TextInput as SystemTextInput,
+    TextStyle,
+    TouchableOpacity,
+    View,
+    ViewStyle
+} from 'react-native';
+import {Colors, Indents, TextSize} from "../constants/styles";
```

Comment:
Code import convention says that first we have libraries than space than local imports :)